### ISSUE TITLE
This feature was added to Meteor Feb 2016

### ISSUE THREAD

**Issue by malhal** [Meteor-Community-Packages/mongo-collection-instances#32]
As of Feb 2016 it is now possible to look up an existing collection, finally!!!

https://github.com/meteor/meteor/pull/5845 (initial attempt)
https://github.com/meteor/meteor/pull/6160 (fix)

Works like this:

`Meteor.connection._stores['tasks']._getCollection()`

**Comment by dburles** [comment]
Hey @malhal thanks for letting me know, strange that no one mentioned it to me earlier

**Comment by malhal** [comment]
So now we know they added it as a private API so that you could make use of it in your public API...will you be using it? 😉

**Comment by dburles** [comment]
Yep, I'll try to update it sometime this week, though it looks like it won't be backwards compatible.

**Comment by malhal** [comment]
Maybe there is a way to implement it differently depending on Meteor's version number?

**Comment by dburles** [comment]
`api.versionsFrom` should take care of that. I'll also publish this current version as 1.0 (I should have done that a while back) then I'll release the newer one as 2.0.

**Comment by malhal** [comment]
Nice! By the way I should mention so far I've only used this on the client so not sure about the server.

**Comment by dburles** [comment]
Hmm, would you mind finding out if that's the case back over on the PR?

**Comment by grsabreu** [comment]
For the server you can fetch them like:

```
Package['mongo].MongoInternals.defaultRemoteCollectionDriver().open('colName')
```

**Comment by matteodem** [comment]
as in with import / export: 

```
import { MongoInternals } from 'meteor/mongo'

MongoInternals.defaultRemoteCollectionDriver().open('colName')
```

**Comment by thearabbit** [comment]
Good, Now I found it 💯

**Comment by dburles** [comment]
@StorytellerCZ I vaguely remember looking into this, granted it was over 5 years ago. I recall though that it wasn't ideal as there was no way to get collections across different connections, unlike the way this package currently works. Though there still might be a potential enhancement here.

**Comment by StorytellerCZ** [comment]
That is my conclusion as well. I will look into this in the future.
